app/models/waste_carriers_engine/finance_details.rb

```diff
@@ -26,6 +26,15 @@ def self.new_renewal_finance_details(transient_registration, method, current_use
       finance_details
     end
 
+    # This amounts are used in a number of finance operations.
+    def amount_in_excess
```

Comment:
My feedback is that I think the names of these methods should be more about what they are reporting and less about the reason we have added them. Spoke with @irisfaraway because I'm useless at naming things, and she has what I think are great suggestions.

```suggestion
    def overpaid_balance
```